docs/protocol.md

```diff
@@ -105,63 +105,78 @@ In the table below:
-> $`J_{ekem,pk}`$ to encrypt its message. The journalist, in turn, authenticates
-> itself using the new long-term key $`J_{dh,pk}`$ and relies on the source's
-> long-term keys $`S_{dh,pk}`$ and $`S_{kem,pk}`$ to encrypt messages back to the
-> source securely. (Maier §5.2)
+| Syntax                                                    | Description                                                          |
+| --------------------------------------------------------- | -------------------------------------------------------------------- |
+| $`h \gets \text{Hash}(m)`$                                | Hash message $m$ to digest $h$                                       |
+| $`k \Vert k_1 \Vert \dots \Vert k_n \gets \text{KDF}(m)`$ | Derive one or more keys $k$ from a message $m$                       |
+| $`\sigma \gets^{\$} \text{Sign}(sk, m)`$                  | Sign a message $m$ with the private key $sk$                         |
+| $`b \in \{0,1\} \gets \text{Vfy}(pk, m, \sigma)`$         | Verify a message $m$ and a signature $\sigma$ with a public key $pk$ |
+| $` g^x \gets \text{DH(g, x)}`$                            | Diffie-Hellman exponentiation of private component $x$               |
+| $`r \gets^{\$} \text{Rand}()`$                            | Generate a random value                                              |
+| $`mp \gets \text{Pad}(m)`$                                | Pad a message $m$ to a constant size[^1]                             |
+| $`\varepsilon`$                                           | The empty string                                                     |
+
+### `AKEM`: Authenticated KEM
+
+For $\text{AKEM} = \text{DHKEM}(\text{Group}, \text{KDF})$ with:
+
+- $\text{Group} =$ [X25519][RFC 9180 §7.1]
+- $\text{KDF} =$ [HKDF-SHA256][RFC 9180 §7.1]
+
+| Syntax                                         | Description                                                                                                                                                                        |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| $`(sk, pk) \gets^{\$} \text{KGen}()`$          | Generate keys; for DH-AKEM, $(sk, pk) = (x, \text{DH}(g, x)) = (x, g^x)$                                                                                                           |
+| $`(c, K) \gets^{\$} \text{AuthEncap}(sk, pk)`$ | Encapsulate a ciphertext $c$ and a shared secret $K$ using a (sender's) private key $sk$ and a (receiver's) public key $pk$; for DH-AKEM, $(c, K) = (pkE, K) = (pk, K) = (g^x, K)$ |
+| $`K \gets \text{AuthDecap}(sk, pk, c)`$        | Decapsulate a shared secret $K$ using a (receiver's) private key $sk$, a (sender's) public key $pk$, and a ciphertext $c$; for DH-AKEM, $c = pkE$                                  |
```

Comment:
(nit: putting sender and receiver for clarification?)

$`(c, K) \gets^{\$} \text{AuthEncap}(sk_S, pk_R)`$ 
$`K \gets^{\$} \text{AuthDecap}(sk_R, pk_S, c)`$